.ansible-lint

```diff
@@ -0,0 +1,7 @@
+---
+exclude_paths:
```

Comment:
@jerome-caucat IMHO we should ignore all linting errors on 3rd party roles. So all roles not prefixed with `laprimaire.` must be excluded. Doable with a regexp/whitelist?

Reply:
We cannot use glob or regex in the config's `exclude_paths`.

However we can select the files to lint in the pre-commit hook and the GitHub Action.

The config's `exclude_paths` is useful when manually running ansible-lint, but we shouldn't have to do this anymore with the pre-commit hook. And when developing we can still lint a single file.

So should I remove this `exclude_paths`?

Reply:
No we will exclude 3rd party roles on a case by case basis.